plugins/copr.py

```diff
@@ -188,6 +192,9 @@ def _read_config_item(self, config, hub, section, default):
         except (NoOptionError, NoSectionError):
             return default
 
+    def _user_warning_before_prompt(self, text):
+        sys.stderr.write("\n{0}\n\n".format(text.strip()))
```

Comment:
So the leading `"/n"` makes it print an empty line at the beginning of the first block of text and also results in there being two line-breaks between the first block of text and the potential second one (the dependency warning). I would remove it, making it not print an empty line at the beginning and resulting in a single empty line between the two blocks of text, which IMO looks better?

Reply:
Ok, I'm a bit distracted by `Updating Subscription Management repositories.` _stdout_ text :-( which made me think the initial space is needed.  Good catch.

Reply:
Oh, I see. I'm testing on Fedora :slightly_smiling_face:

Reply:
Me too.